cluster/gce/config-test.sh

```diff
@@ -319,9 +319,9 @@ NODE_PROBLEM_DETECTOR_TAR_HASH="${NODE_PROBLEM_DETECTOR_TAR_HASH:-}"
 NODE_PROBLEM_DETECTOR_RELEASE_PATH="${NODE_PROBLEM_DETECTOR_RELEASE_PATH:-}"
 NODE_PROBLEM_DETECTOR_CUSTOM_FLAGS="${NODE_PROBLEM_DETECTOR_CUSTOM_FLAGS:-}"
 
-CNI_STORAGE_PATH="${CNI_STORAGE_PATH:-https://storage.googleapis.com/kubernetes-release/network-plugins}"
 CNI_VERSION="${CNI_VERSION:-}"
 CNI_SHA1="${CNI_SHA1:-}"
+CNI_STORAGE_PATH="${CNI_STORAGE_PATH:-https://github.com/containernetworking/plugins/releases/download/${CNI_VERSION}}"
```

Comment:
If CNI_VERSION isn't explicitly set it won't be readable here. Instead this shouldn't include CNI_VERSION and we need to account for it below
```suggestion
CNI_STORAGE_PATH="${CNI_STORAGE_PATH:-https://github.com/containernetworking/plugins/releases/download/}"
```